node/src/database/rocksdb.rs

```diff
@@ -7,130 +7,350 @@
 use super::{Registry, Tx, DB};
 use anyhow::Result;
 
+use dusk_consensus::commons::Block;
+use dusk_consensus::messages::Serializable;
+use rocksdb_lib::{
+    ColumnFamily, DBAccess, DBCommon, DBWithThreadMode, MultiThreaded,
+    OptimisticTransactionDB, OptimisticTransactionOptions, Options,
+    ReadOptions, SnapshotWithThreadMode, WriteOptions,
+};
+use std::{marker::PhantomData, path::Path, sync::Arc};
```

Comment:
Please reformat this `use`

```suggestion
use std::marker::PhantomData;
use std::path::Path;
use std::sync::Arc;
```

Reply:
Done. 
NB: That's added usually by the IDE.  If we don't have a custom linter rule to catch this, it'll appear again.